frontend/server/src/Controllers/Problem.php

```diff
@@ -4733,13 +4712,19 @@ public static function getProblemEditDetailsForSmarty(
                 'statusSuccess' => false,
                 'admins' => $admins,
                 'groupAdmins' => $groupAdmins,
-                'tags' => $tags,
                 'problemLevel' => \OmegaUp\DAO\ProblemsTags::getProblemLevel(
                     $problem
                 ),
+                'selectedPublicTags' => \OmegaUp\DAO\ProblemsTags::getTagsForProblem(
+                    $problem,
+                    true /* public */
```

Comment:
nit:

```suggestion
                    /*$public=*/true
```

en todos los demás lugares se pone el nombre del argumento antes. igual abajo.